docs/accessibility.md

```diff
@@ -66,6 +66,19 @@ iOS In the above example, VoiceOver will read the hint after the label, if the u
 
 Android In the above example, TalkBack will read the hint after the label. At this time, hints cannot be turned off on Android.
 
+### `accessibilityLanguage` <div class="label ios">iOS</div>
+
+Using the `accessibilityLanguage` property the screen reader will understand which language to use while reading the element's **label**, **value** and **hint**. The provided string value must follow the [BCP 47 specification](https://www.rfc-editor.org/info/bcp47).
```

Comment:
**By** using the `accessibilityLanguage` property, the screen reader will ... 

^^ I think it will make the sentence easy to read. 😃

Reply:
I like it, thank you! 🙂 Added here: https://github.com/facebook/react-native-website/pull/2970/commits/56f6f28aca62c9a109cba5cc7fe40af36d4202be